features/filter_by_pubkey.feature

```diff
@@ -0,0 +1,149 @@
+    And the results should contain items from "npub1alice789xyz..." and "npub1bob456def..."
+    But the results should not contain items from "npub1charlie123abc..."
+
+  Scenario: Combine pubkey filter with item type filter
+    When I filter items by pubkey "npub1alice789xyz..."
+    And I filter by item type "PROJECT"
+    Then I should see 1 item in the results
+    And the result should be:
+      | item_id | type    | title              | proposer_npub       |
+      | proj_1  | PROJECT | Cross-Chain Bridge | npub1alice789xyz... |
+
+  Scenario: Filter returns empty results when no items match
+    When I filter items by pubkey "npub1nonexistent999..."
+    Then I should see 0 items in the results
+    And I should see a message "No items found for the specified publisher"
+
+  Scenario: Invalid npub format returns validation error
+    When I filter items by pubkey "invalid-npub-format"
+    Then I should receive a validation error
+    And the error should indicate "Invalid Nostr public key format"
+    And no filtering should be applied
+
+  Scenario: Filter with pagination for large result sets
+    Given "npub1alice789xyz..." has published 150 items
+    When I filter items by pubkey "npub1alice789xyz..."
+    And I request page 1 with 50 items per page
+    Then I should see 50 items in the current page
+    And I should see pagination info:
+      | Field        | Value |
+      | total_items  | 150   |
+      | current_page | 1     |
+      | total_pages  | 3     |
+      | per_page     | 50    |
+
+  Scenario: Filter updates in real-time when new items are published
+    Given I am viewing filtered results for "npub1alice789xyz..."
+    And the results show 3 items
+    When "npub1alice789xyz..." publishes a new TASK "Backend Integration"
+    Then the filtered results should automatically update
+    And I should now see 4 items in the results
+    And the new task should appear in the list
+
+  Scenario: Clear pubkey filter to view all items
+    Given I have filtered items by pubkey "npub1alice789xyz..."
+    And I see 3 filtered results
+    When I clear the pubkey filter
+    Then I should see all 7 items
+    And items from all publishers should be visible
+
+  Scenario Outline: Filter by pubkey across different item types
+    When I filter items by pubkey "<pubkey>"
+    And I filter by item type "<item_type>"
+    Then I should see <count> items in the results
+
+    Examples:
+      | pubkey                | item_type | count |
+      | npub1alice789xyz...   | PROJECT   | 1     |
+      | npub1alice789xyz...   | TASK      | 1     |
+      | npub1alice789xyz...   | BOUNTY    | 1     |
+      | npub1bob456def...     | PROJECT   | 1     |
+      | npub1bob456def...     | TASK      | 1     |
+      | npub1bob456def...     | BOUNTY    | 0     |
+      | npub1charlie123abc... | PROJECT   | 1     |
+      | npub1charlie123abc... | TASK      | 0     |
+      | npub1charlie123abc... | BOUNTY    | 1     |
+
+  # UI/UX Scenarios
```

Comment:
I like that you explicitly mark these as UI tests. I'm not 100% sure how much we want to specify these in our feature files, but since all of these are opt-in for testing with the `@automated` tag, I'm fine with leaving things in unless there's a good reason not to. If we do hook these up with automated tests, I'm guessing we would just use [Playwright](https://playwright.dev/).